lib/loaders/eslint.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * This file is part of the Symfony Webpack Encore package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+const loaderFeatures = require('../features');
+
+/**
+ * @param {WebpackConfig} webpackConfig
+ * @return {Object} of options to use for eslint-loader options.
+ */
+module.exports = {
+    getOptions(webpackConfig) {
+        loaderFeatures.ensurePackagesExist('eslint');
+
+        const eslintLoaderOptions = {
+            parser: 'babel-eslint',
+            emitWarning: true,
+            'import/resolver': {
+                webpack: {
+                    config: 'webpack.config.js'
+                }
+            }
```

Comment:
Can you explain this config? What does this do exactly?

Reply:
because you can define additional alias in webpack, this import/Resolver allows eslint to verify that libraries/files exist.

Reply:
So, with `config: 'webpack.config.js'`, we are telling `eslint` that it should internally require our `webpack.config.js` file to learn about the aliases? If so, I think that would cause an error - as the Encore "runtime" config wouldn't be defined. Are we sure this works? If we can add a test case for it in `functional.js`, I would be SUPER happy :)

Reply:
Ping! This is my main concern / question about this PR. I simply don't understand what this config does, so I'm uncomfortable merging. Can you guide me @pinoniq?

Reply:
Yes! I think this is exactly what we should do. So, once you've removed this config and the reference to the package, I think this is ready :).